api/src/utils/get-address.test.ts

```diff
@@ -47,17 +47,20 @@ describe('getAddress', async () => {
 		});
 
 		expect(getAddress(server)).toBe(`${serverHost}:${serverPort}`);
+		server.close();
```

Comment:
Since all tests use server.close(), we could refactor this with an afterEach:

```ts
afterEach(() => {
  server?.close();
});
```

Reply:
I don't think so as `server` is locally scoped, unless we refactor it out

Reply:
Yes, that's why we'd need to refactor it by declaring `let server: http.Server` at the describe block level, then assign to it in each test (instead of `const server = ...`). This way the afterEach can access it :)